python/pynessie/commands/_branch_tag_handlers.py

```diff
@@ -99,16 +101,38 @@ def _handle_json_output(input_data: list, ref_name: str) -> str:
     return ReferenceSchema().dumps(input_data, many=True)
 
 
-def _handle_normal_output(input_data: list, verbose: bool, default_branch: str) -> str:
+def _handle_normal_output(input_data: list, verbose: bool, default_branch: str, show_additional_info: bool = False) -> str:
     output = ""
     max_width = max((len(i.name) for i in input_data), default=0)
     for x in input_data:
+        additional_info = ""
+        if show_additional_info and x.metadata:
+            num_commits_ahead = x.metadata.num_commits_ahead if x.metadata.num_commits_ahead else "?"
+            num_commits_behind = x.metadata.num_commits_behind if x.metadata.num_commits_behind else "?"
+            num_total_commits = x.metadata.num_total_commits if x.metadata.num_total_commits else "?"
+            common_ancestor_hash = x.metadata.common_ancestor_hash if x.metadata.common_ancestor_hash else "?"
+            commit_msg = (
+                x.metadata.commit_meta_of_head.message[:35] + "..."
+                if len(x.metadata.commit_meta_of_head.message) >= 35
+                else x.metadata.commit_meta_of_head.message
+            )
+            head = (
+                "[author: {} | commitTime: {} | msg: {}]".format(
+                    x.metadata.commit_meta_of_head.author, x.metadata.commit_meta_of_head.commitTime, commit_msg
+                )
+                if x.metadata.commit_meta_of_head
+                else "?"
+            )
+            additional_info = " [total commits: {} | commits behind: {} | commits ahead: {} | ancestor: {} | HEAD: {}]".format(
```

Comment:
Can we use here `f"{}"` instead of `.format`? I also find `f"{}"` syntax cleaner 😄

Reply:
yep good idea, updated.